### ISSUE TITLE
Integrate with FuseSoC

### ISSUE THREAD

**Issue by mithro** [google/verible#1]
https://github.com/olofk/fusesoc
> FuseSoC is an award-winning package manager and a set of build tools for HDL (Hardware Description Language) code.
> 
> Its main purpose is to increase reuse of IP (Intellectual Property) cores and be an aid for creating, building and simulating SoC solutions.
> 
> FuseSoC makes it easier to
> * reuse existing cores
> * create compile-time or run-time configurations
> * run regression tests against multiple simulators
> * Port designs to new targets
> * let other projects use your code
> * set up continuous integration

Would be good to have FuseSoc able to use Verible as a linter and code formatter.

**Comment by mithro** [comment]
@olofk

**Comment by olofk** [comment]
Definitely. Adding a new Edalize backend for the linter and formatter should be straightforward

**Comment by fangism** [comment]
FYI @msfschaffner @sjgitty
Already in use? done or needs more work?

**Comment by imphil** [comment]
Verible lint is done in https://github.com/olofk/edalize/pull/100, verible format isn't done yet.
